app/emoji.go

```diff
@@ -224,6 +224,20 @@ func (a *App) SearchEmoji(name string, prefixOnly bool, limit int) ([]*model.Emo
 	return a.Srv.Store.Emoji().Search(name, prefixOnly, limit)
 }
 
+// GetEmojiStaticUrl returns a relative static URL for system default emojis,
+// and the API route for custom ones. Errors if not found or if custom and deleted.
+func (a *App) GetEmojiStaticUrl(emojiName string) (string, *model.AppError) {
+	if id, found := model.GetSystemEmojiId(emojiName); found {
+		return fmt.Sprintf("/static/emoji/%s.png", id), nil
```

Comment:
This needs to take into account a potential subpath in the `SiteURL`, e.g. something like:
```go
subpath, _ := utils.GetSubpathFromConfig(c.App.Config())
return path.Join(subpath, "static", "emoji", fmt.Sprintf("%s.png", id), nil
```